src/test/java/com/Query/query/Controller/AlunoControllerTest.java

```diff
@@ -19,7 +19,7 @@
 	private MockMvc mockMvc;
 
 	@Test
-	private void retornaAlunoStatus200paraUmParametro() throws Exception {
+	public void retornaAlunoStatus200paraUmParametro() throws Exception {

```

Comment:
Esqueci de comentar que a partir do JUnit 5 não é mais preciso colocar a visibilidade "public". Aplicar essa alteração em todos os métodos e declarações das classes de teste.
```suggestion
	void retornaAlunoStatus200paraUmParametro() throws Exception {
```

Reply:
feito